deezer/client.py

```diff
@@ -185,6 +185,27 @@ def get_object(
                     object_t, object_id
                 )
             )
+
+        # fix deezer's truncation of items (e.g. tracks on large playlists)
+        if (
+            (
+                relation == "tracks"
+                or relation == "fans"
+                or relation == "albums"
+                or relation == "artists"
+                or relation == "playlists"
+            )
+            and "next" in json
+            and "limit" not in kwargs
+        ):
+            new_json = json
+            while "next" in new_json:
+                response = self.session.get(new_json["next"])
+                new_json = response.json()
+                json["data"].extend(new_json["data"])
```

Comment:
This is bypassing the error handling that we have above. What happens if we get an error in the middle? Could we get throttled by Deezer in this section?

Reply:
I'll fix this. What do you suggest happens if we get throttled? My justification for fixing this in the first place was that "if you ask for `<playlist>`'s tracks, you want *all* the tracks", but what should happen if we get cut off halfway through (or whenever)?